src/components/pages/room.js

```diff
@@ -190,7 +190,7 @@ export default function Room({ PageTemplate, initialState }) {
               </Tooltip>
             </Fragment>
           )}
-          <span>{startsOn ? formatDate(startsOn) : t('notScheduled')}</span>
+          <span>{startsOn ? formatDate(startsOn) : ''}</span>
```

Comment:
I think you wanted to not make this change with this ticket, based on your JIRA comment, right?